### ISSUE TITLE
Gulp syntax error: Unexpected token import

### ISSUE THREAD

**Issue by valos** [i18next/i18next-parser#86]
- Is this a bug or a feature request?
Bug

- What version are you running (`0.x` or `1.x`)? 
1.0.0-beta9

- Which runner (cli, gulp...)?
gulp

## What I'm trying to do?

Extract translations from JS and HTML (pseudo handlebars templates) files

## How am I doing it?

Here is my config (extracted from gulpfile.js):

```
{
  locales: ['en', 'fr'],
  output: '../locales',
  functions: ['__'],
  createOldCatalogs: false
}
```

The command I'm running:

```
gulp i18next
```

## What was the result?

An error:

```
.../node_modules/i18next-parser/src/index.js:1
(function (exports, require, module, __filename, __dirname) { import { dotPathToHash, mergeHashes, populateHash } from './helpers'
                                                              ^^^^^^

SyntaxError: Unexpected token import
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:616:28)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (.../gulpfile.js:34:17)
```

## What was I expecting?

My translations in output folder.

**Comment by valos** [comment]
I got a new error:

```
$ gulp i18next
[09:41:08] Using gulpfile gulpfile.js
[09:41:08] Starting 'i18next'...
[09:41:08] 'i18next' errored after 7.05 ms
[09:41:08] TypeError: i18next is not a function
    at Gulp.<anonymous> (gulpfile.js:38:15)
    at module.exports (node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (node_modules/orchestrator/index.js:214:10)
    at Gulp.Orchestrator.start (node_modules/orchestrator/index.js:134:8)
    at /usr/lib/node_modules/gulp/bin/gulp.js:129:20
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:188:16)
```

### Versions
```
$ gulp -v
[09:58:48] CLI version 3.9.1
[09:58:48] Local version 3.9.1

$ nodejs -v
v8.10.0
```

### Gulp code
```
const i18next = require('i18next-parser');

gulp.task('i18next', function() {
    gulp.src(['app/pages/*', 'app/js/*', 'app/index.html'])
        .pipe(i18next({
            locales: ['en', 'fr'],
            functions: ['__'],
            output: '../locales',
            createOldCatalogs: false
        }))
        .pipe(gulp.dest('app/locales'));
});
```

**Comment by karellm** [comment]
This was fixed and test in `1.0.0-beta11`. Please note that the README was also updated. Here is the proper way to make it work with gulp:

```js
const gulp = require('gulp');
const i18next = require('i18next-parser');

gulp.task('i18next', function() {
    gulp.src(['app/pages/*'])
        .pipe(new i18next(...))
        .pipe(gulp.dest('./'));
});
```
